HW1 Completed

schoure98 opened the pull request:
**Exercise 01**: 
selectors.md file contains answers to CSS diner game. 

**Exercise 02**: 
palindrome.html file displays the expected palindrome output given in HW1. Bootstrap classes are used for styling. 
palindrome.js file contains the logic to check if the given input is a valid palindrome or not. It also checks, if the input is a positive numeric value and not the text value. 

Please review my code and let me know the required changes.

schoure98 committed:
HW1 Completed

juhiaugustine commented on hw1/02-palindrome.html:
```diff
@@ -1,27 +1,28 @@
+			<section class="bg-light rounded p-5">
+				<h1>Exercise 02 - Palindrome</h1>
+				<div>
+					<label>Enter a positive number:</label>
```
Since the label is not associated with the input element, this might probably cause error in the WAVE analysis. Can you verify it once using WAVE from your end.

schoure98 replied:
Thanks for noticing. I have added for to the label now.

juhiaugustine commented on hw1/02-palindrome.js:
```diff
@@ -1,3 +1,27 @@
+	}
+};
+
+elem.addEventListener("input", handleInput);
```
This code looks neat and well optimized!

schoure98 replied:
Thanks Juhi :)

queoui commented:
Hey Shreya,
One thing I noticed when using your palindrome card is that the "No, Try again." does not go away when you delete the numbers and the box is empty again. You might want to make it so any response goes away when the box is empty. Also, a single digit should be a palindrome but when I only put in '1' it says that it is not a palindrome, a simple if statement would fix this. Otherwise great job!

queoui commented on hw1/02-palindrome.js:
```diff
@@ -1,3 +1,27 @@
+const output = document.getElementById("output");
 
-elem.addEventListener('input', handleInput);
+handleInput = () => {
```
Nice work using an anonymous function here, the AirBnB style guide suggests slightly different syntax however. Might want to look that up. I used a linter with my project and it helped with some of the more minor style syntax errors.

schoure98 replied:
Thanks for letting me know. I have updated the functions in latest commit.

schoure98 committed:
added palindromeChecker function, null condition input

schoure98 committed:
updated function according to style guide

schoure98 commented:
@queoui thanks for noticing. I have added the if condition for null input. And also added, palindromeChecker function and now able to check if palindrome for single number input.
